org.eclipse.buildship.core/META-INF/MANIFEST.MF

```diff
@@ -25,6 +25,7 @@ Export-Package: org.eclipse.buildship.core;x-friends:="org.eclipse.buildship.ui"
  org.eclipse.buildship.core.configuration;x-friends:="org.eclipse.buildship.ui",
  org.eclipse.buildship.core.console;x-friends:="org.eclipse.buildship.ui",
  org.eclipse.buildship.core.event;x-friends:="org.eclipse.buildship.ui",
+ org.eclipse.buildship.core.extensions,
```

Comment:
Let's find a better name for this. We don't want to throw all extensions into one package. The package name should reflect the usage. Something like `invocation` or `synchronization`.